### PR TITLE
feat(video-filter): add VxTestAppChecker boilerplate code

### PR DESCRIPTION
**Pull request by a1exwang**
* parse `installOptions` argument for prerequisite check task 
* add an empty VxTestAppChecker 

image

image

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16037812/

### CODE REVIEW

**Comment by a1exwang** [comment]
E2E TEST: n/a

File: packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts [hunk 1]
```diff
@@ -77,6 +87,7 @@ export class PrerequisiteTaskTerminal extends BaseTaskTerminal {
     const res = await checkAndInstallForTask(
       this.args.prerequisites ?? [],
       this.args.portOccupancy,
+      this.args.installOptions,
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts:90]: Can we just use `vxTestApp` as the arg name? IMO, it's clear that each prerequisite just use its name as option name:
``` json
{
    "prerequisites": [
        "AAA",
        "BBB",
        "..."
    ],
    "AAA": {},
    "BBB": {},
    "..."
}
```

Comment (swatDong) [reply packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts:90]: The implementation can convert them to different types of options, but for UX less level is clearer.

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts [hunk 2]
```diff
@@ -107,6 +108,7 @@ const DepsDisplayName = {
   [DepsType.Dotnet]: ".NET Core SDK",
   [DepsType.Ngrok]: "ngrok",
   [DepsType.FuncCoreTools]: "Azure Functions Core Tools",
+  [DepsType.VxTestApp]: "Video Filter Test App",
```
Comment (qinezh) [inline packages/vscode-extension/src/debug/prerequisitesHandler.ts:111]: Let's use `Video Extensibility Test App` to keep consistent.
image

File: packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts (hunk 1 above)
Comment (a1exwang) [reply packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts]: Updated

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts (hunk 2 above)
Comment (a1exwang) [reply packages/vscode-extension/src/debug/prerequisitesHandler.ts]: Updated

**Review by swatDong — APPROVED** [review]

**Review by qinezh — APPROVED** [review]

**Pull request merged**